### ISSUE TITLE
Low performance of motorcyclist and other-ground classes

### ISSUE THREAD

**Issue by JBKnights** [xinge008/Cylinder3D#37]
Hello,

For the pre-trained model included, I noticed that performance on the classes motorcyclist and ground-other are close to zero, while in the paper their reported accuracy is much higher.  Is this because of the lack of point refinement module in this repository, or is there some other explanation as to the significant difference in result here?

**Comment by xinge008** [comment]
Due to the limited number of points of motorcyclist and other-ground, the results on validation set are very poor, even close to zero.

The results reported in the paper are from test set, which is more balanced compared to validation set.
